Client/Utils/MarkupParser/MarkupNode.swift

```diff
@@ -0,0 +1,19 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+public enum MarkupNode: Equatable {
```

Comment:
Doesn't need to be marked public if we're under Client?
```suggestion
enum MarkupNode: Equatable {
```